pages/linux/dnsenum.md

```diff
@@ -0,0 +1,16 @@
+# dnsenum
+
+> Multithreaded Perl script used to enumerate DNS information from a domain.
+> More information: <https://github.com/SparrowOchon/dnsenum2>.
+
+- Enumerate a domain with default settings:
+
+`dnsenum -enum {{example.com}}`
+
+- Enumerate a domain without reverse lookup and save the output to a file:
+
+`dnsenum --noreverse -o {{path/to/file}} {{example.com}}`
+
+- Brute force search on subdomains along with a custom file with subdomains passed as an attribute:
+
+`dnsenum -f {{filename}} -r {{example.com}}`
```

Comment:
```suggestion
- Brute force search on subdomains along with a specific file with subdomains passed as an attribute:

`dnsenum --file {{path/to/file}} --recursion {{example.com}}`
```

Almost the same issue as above.
Mandatory fix: [path format is broken](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#paths).
Mandatory fix: [long option must be used if it's portable](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#option-syntax).